src/game/AI/ScriptDevAI/scripts/kalimdor/darkshore.cpp

```diff
@@ -486,107 +461,59 @@ struct npc_volcorAI : public npc_escortAI
-            case 5:
-                m_creature->SummonCreature(NPC_BLACKWOOD_SHAMAN, aVolcorSpawnLocs[0].m_fX, aVolcorSpawnLocs[0].m_fY, aVolcorSpawnLocs[0].m_fZ, aVolcorSpawnLocs[0].m_fO, TEMPSPAWN_TIMED_OOC_DESPAWN, 20000);
-                m_creature->SummonCreature(NPC_BLACKWOOD_URSA, aVolcorSpawnLocs[1].m_fX, aVolcorSpawnLocs[1].m_fY, aVolcorSpawnLocs[1].m_fZ, aVolcorSpawnLocs[1].m_fO, TEMPSPAWN_TIMED_OOC_DESPAWN, 20000);
-                break;
-            case 6:
-                DoScriptText(SAY_FIRST_AMBUSH, m_creature);
-                break;
-            case 11:
-                m_creature->SummonCreature(NPC_BLACKWOOD_SHAMAN, aVolcorSpawnLocs[2].m_fX, aVolcorSpawnLocs[2].m_fY, aVolcorSpawnLocs[2].m_fZ, aVolcorSpawnLocs[2].m_fO, TEMPSPAWN_TIMED_OOC_DESPAWN, 20000);
-                m_creature->SummonCreature(NPC_BLACKWOOD_URSA, aVolcorSpawnLocs[3].m_fX, aVolcorSpawnLocs[3].m_fY, aVolcorSpawnLocs[3].m_fZ, aVolcorSpawnLocs[3].m_fO, TEMPSPAWN_TIMED_OOC_DESPAWN, 20000);
-            case 13:
-                m_creature->SummonCreature(NPC_BLACKWOOD_URSA, aVolcorSpawnLocs[4].m_fX, aVolcorSpawnLocs[4].m_fY, aVolcorSpawnLocs[4].m_fZ, aVolcorSpawnLocs[4].m_fO, TEMPSPAWN_TIMED_OOC_DESPAWN, 20000);
-                m_creature->SummonCreature(NPC_BLACKWOOD_URSA, aVolcorSpawnLocs[5].m_fX, aVolcorSpawnLocs[5].m_fY, aVolcorSpawnLocs[5].m_fZ, aVolcorSpawnLocs[5].m_fO, TEMPSPAWN_TIMED_OOC_DESPAWN, 20000);
+            case 3:
+                if (m_uiQuestId == QUEST_ESCAPE_THROUGH_STEALTH)
+                {
+                    // Reward Player
+                    if (Player* pPlayer = GetPlayerForEscort())
+                        pPlayer->RewardPlayerAndGroupAtEventExplored(QUEST_ESCAPE_THROUGH_STEALTH, m_creature);
+                    m_creature->GetMotionMaster()->Clear(false, true);
+                    End();
+                    m_creature->ForcedDespawn(1000);
+                }
                 break;
             case 15:
-                DoScriptText(SAY_END, m_creature);
-                if (Player* pPlayer = GetPlayerForEscort())
-                    pPlayer->RewardPlayerAndGroupAtEventExplored(QUEST_ESCAPE_THROUGH_FORCE, m_creature);
-                SetEscortPaused(true);
-                m_creature->ForcedDespawn(10000);
-                break;
-            // Quest 995 waypoints
-            case 16:
-                m_creature->HandleEmote(EMOTE_ONESHOT_BOW);
-                break;
-            case 17:
-                if (Player* pPlayer = GetPlayerForEscort())
-                    DoScriptText(SAY_ESCAPE, m_creature, pPlayer);
-                break;
-            case 18:
-                DoCastSpellIfCan(m_creature, SPELL_MOONSTALKER_FORM);
-                break;
-            case 24:
                 if (Player* pPlayer = GetPlayerForEscort())
                     pPlayer->RewardPlayerAndGroupAtEventExplored(QUEST_ESCAPE_THROUGH_STEALTH, m_creature);
+                m_creature->GetMotionMaster()->Clear(false, true);
```

Comment:
Also call move idle after this otherwise that might lead to problems i think

Reply:
Never had problem while testing but i added it for safty